lib/ids.js

```diff
@@ -48,11 +59,41 @@ const generateId = (length) => {
   return completedId;
 };
 
+const getIdLengthDoc = db =>
+  new Promise((resolve, reject) =>
+    db.medic.get(ID_LENGTH_DOC_ID, (err, result) => {
+      if (err) {
+        if (err.statusCode !== 404) {
+          return reject(err);
+        } else {
+          result = {
+            _id: ID_LENGTH_DOC_ID
+          };
+
+          result[ID_LENGTH_PARAM] = INITIAL_ID_LENGTH;
+        }
+      }
+
+      resolve(result);
+    }));
+
+const putIdLengthDoc = (db, idLengthDoc) =>
+  new Promise((resolve, reject) =>
+    db.medic.put(idLengthDoc, err => {
+      // We're OK with a 409, because we're going to presume this is happening
+      // because a human edited it to suite their needs, and their write is more
+      // important than ours.
+      if (err && err.statusCode !== 409) {
+        reject(err);
+      } else {
+        resolve();
+      }}));
+
```

Comment:
They currently are being essentially ignored yeah. I'll add a `console.log` about it.